src/field3d.imageio/field3doutput.cpp

```diff
@@ -278,8 +278,8 @@ Field3DOutput::put_parameter (const std::string &name, TypeDesc type,
                 Strutil::split (format_list, format_prefixes, ",");
                 format_prefixes_initialized = true;
             }
-            for (size_t i = 0, e = format_prefixes.size();  i < e;  ++i)
-                if (Strutil::iequals (prefix, format_prefixes[i]))
+            for (const auto & f : format_prefixes)
```

Comment:
Yes. In most cases, I just let it be. Though in cases where I chose to go and correct its choice of variable names, I also "fixed" the spacing.

If you feel strongly about it, I can fix them all. But I was prepared to just accept the free work that clang-tidy did for me and not nit pick a slightly different, but equally readable, spacing than I would have used if I'd made the changes by hand.